scripts/keycloak/full-kube-clowder-realm-prod.json

```diff
@@ -0,0 +1,1914 @@
+{
```

Comment:
Maybe just keep it here for now until we figure out a more generic way to set this. i.e. pass it in as a file to the helm chart